marconi-chain-index/src/Marconi/ChainIndex/Indexers.hs

```diff
@@ -626,9 +626,9 @@ mkIndexerStream' f coordinator =
       waitQSemN _barrier _indexerCount
       pure c'
 
-    -- give 3 minutes (180s) for a step to finish
+    -- give 30 minutes (1800s) for a step to finish
```

Comment:
Isn't that too much?

Reply:
3 minutes wasn't enough on mainnet. Made it fail. Still failed at 5min, so I thought to just put a large enough number. Then again, something between 15min or 30min, not much difference. I don't mind 15min though.

Reply:
I wouldn't thought it would take that long. We should definitely push it to a separate thread. We can keep it to 30 minutes atm and we'll work on it in another ticket.